yellow-paper/docs/cryptography/protocol-overview.md

```diff
@@ -0,0 +1,113 @@
+
+Rollup-side, each "step" in the IVC scheme is a Honk proof, which are recursively verified. As a result, no protoocols other than Honk are required to execute rollup-side IVC.
+
+We perform one layer of "proof-system compression" in the rollup. The final proof of block-correctness is constructed as a Honk proof. An UltraPlonk circuit is used to verify the correctness of the Honk proof, so that the proof that is verified on-chain is an UltraPlonk proof (verification gas costs are lower for UltraPlonk vs Honk).
+
+The following sections list the protocol components required to implement client-side IVC.
+
+## Protogalaxy
+
+The [Protogalaxy](https://eprint.iacr.org/2023/1106) protocol defines a folding scheme that enables instances of a relation to be folded into a single instance of a "relaxed" form of the original relation.
+
+It is a variant of [Protostar](https://eprint.iacr.org/2023/620). Unlike Protostar, Protogalaxy enables multiple instances to be efficiently folded into the same accumulator instance.
+
+The Protogalaxy protocol is split into two subprotocols, each modelled as interactive protocols between a Prover and a Verifier.
+
+#### Protogalaxy Fold
+
+The "Fold" Prover/Verifier validates that `k` instances of a defined relation (in our case the Honk relation) have been correctly folded into an accumulator instance.
+
+#### Protogalaxy Decider
+
+The "Decider" Prover/Verifier valiate whether an accumulator instance correctly satisfies the accumulator relation.
+
+## Goblin Plonk
+
+[Goblin Plonk](https://hackmd.io/@aztec-network/BkGNaHUJn/%2FGfNR5SE5ShyXXmLxNCsg3g) is a computation delegation scheme that improves Prover performance when evaluating complex algorithms.
+
+In the context of an IVC scheme, Goblin Plonk enables a Prover to defer non-native group operations required by a Verifier algorithm, across multiple recursive proofs, to a single step evaluated at the conclusion of the IVC Prover algorithm.
+
+Goblin Plonk is composed of three subcomponents:
+
+#### Transcript Aggregation Subprotocol
+
+This subprotocol aggregates deferred computations from two independent instances, into a single instance
+
+#### Elliptic Curve Virtual Machine (ECCVM) Subprotocol
+
+The ECCVM is a Honk circuit with a custom circuit arithmetisation, designed to optimally evaluate elliptic curve arithmetic computations that have been deferred. It is defined over the Grumpkin elliptic curve
+
+#### Translator Subprotocol
+
+The Translator is a Honk circuit with a custom circuit arithmetisation, designed to validate the input commitments of an ECCVM circuit align with the delegated computations described by a Goblin Plonk transcript commitment
+
+## Plonk Data Bus
+
+The [Plonk Data Bus](https://aztecprotocol.slack.com/files/U8Q1VAX6Y/F05G2B971FY/plonk_bus.pdf) protocol enables efficient data transfer between two Honk instances within a larger IVC protocol.
```

Comment:
A sentence about why this is needed would be good

Reply:
done